modules/oadp-1-5-0-release-notes.adoc

```diff
@@ -0,0 +1,264 @@
+
+With this update, `volumeSnapshotContents` CR are restored even when the `includedNamesapces` field is used in `restoreSpec`. As a result, an application pod is in a `Running` state after restore. 
+
+link:https://issues.redhat.com/browse/OADP-5939[OADP-5939]
+
+.OADP operator successfully creates bucket on top of AWS
+
+Previously, the container was configured with the `readOnlyRootFilesystem: true` setting for security, but the code attempted to create temporary files in the `/tmp` directory using the `os.CreateTemp()` function. Consequently, while using the AWS STS authentication with the Cloud Credential Operator (CCO) flow, {oadp-short} failed to create temporary files that were required for AWS credential handling with the following error:
+[source,terminal]
+----
+ERROR unable to determine if bucket exists. {"error": "open /tmp/aws-shared-credentials1211864681: read-only file system"}
+----
+With this update, the following changes are added to address this issue:
+
+* A new `emptyDir` volume named `tmp-dir` to the controller pod specification.
+* A volume mount to the container, which mounts this volume to the `/tmp` directory.
+* For security best practices, the `readOnlyRootFilesystem: true` is maintained.
+* Replaced the deprecated `ioutil.TempFile()` function with the recommended `os.CreateTemp()` function.
+* Removed the unnecessary `io/ioutil` import, which is no longer needed.
+
+link:https://issues.redhat.com/browse/OADP-6019[OADP-6019]
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12462673[OADP 1.5.0 resolved issues] in Jira.
+
+
+[id="known-issues-1-5-0_{context}"]
+== Known issues
+
+.Kopia does not delete the artifacts after backup expiration
+
+Even after deleting a backup, Kopia does not delete the volume artifacts from the `${bucket_name}/kopia/${namespace}` on the S3 location after the backup expired. To ensure that {oadp-first} functions properly, the data is not deleted, and it exists in the `/kopia/` directory, for example:
+
+* `kopia.repository`: Main repository format information such as encryption, version, and other details. 
+* `kopia.blobcfg`: Configuration for how data blobs are named. 
+* `kopia.maintenance`: Tracks maintenance owner, schedule, and last successful build. 
+* `log*`: Log blobs. 
+
+link:https://issues.redhat.com/browse/OADP-5131[OADP-5131]
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12472334[OADP 1.5.0 known issues] in Jira.
+
+[id="deprecated-features-1-5-0_{context}"]
+== Deprecated features
+
+.The `configuration.restic.podConfig.resourceAllocations` specification field has been deprecated
+
+With {oadp-first} 1.5.0, the `configuration.restic.podConfig.resourceAllocations` specification field has been deprecated. Use the `nodeAgent` section with the `uploaderType` field set to `Kopia` instead of `Restic`.
```

Comment:
```suggestion
With {oadp-first} 1.5.0, the `configuration.restic` specification field has been deprecated. Use the `nodeAgent` section with the `uploaderType` field set to `Kopia` instead of `Restic`.
```